Wonder updates

MeuhV1 opened the pull request:
Strategy texts for all new wonders
Changed Burj Khalifa effect: Desert tiles in city radius base yield is at least 2 food and 2 commerce
Some wonders provide different GPP:
- The Cristo Redentor: Great Engineer -> Great Prophet
- Eiffel Tower: Great Merchant -> Great Artist
- Sagrada Familia: Great Engineer -> Great Artist
- Salt Cathedral: Great Engineer -> Great Merchant
- Shalimar Gardens: Great Engineer -> Great Artist
- Trafalgar Square: Great Merchant -> Great Statesman
Some minor code updates

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -2901,6 +2901,38 @@
+	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_BABYLONIAN_GARDEN_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Garden[COLOR_REVERT], the Babylonian replacement for the [COLOR_BUILDING_TEXT]Ampitheatre[COLOR_REVERT], provides bonus health.</English>
```
By the way, it now replaces the Arena.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -75,11 +83,11 @@
-		<German>[COLOR_BUILDING_TEXT]Borobudur[COLOR_REVERT] allows you to hire three [COLOR_HIGHLIGHT_TEXT]Priest[COLOR_REVERT] specialists and adds additional production to all of your [COLOR_HIGHLIGHT_TEXT]Priest[COLOR_REVERT] specialists. It also increases the odds of its city generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</German>
-		<Italian>[COLOR_BUILDING_TEXT]Borobudur[COLOR_REVERT] allows you to hire three [COLOR_HIGHLIGHT_TEXT]Priest[COLOR_REVERT] specialists and adds additional production to all of your [COLOR_HIGHLIGHT_TEXT]Priest[COLOR_REVERT] specialists. It also increases the odds of its city generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</Italian>
-		<Spanish>[COLOR_BUILDING_TEXT]Borobudur[COLOR_REVERT] allows you to hire three [COLOR_HIGHLIGHT_TEXT]Priest[COLOR_REVERT] specialists and adds additional production to all of your [COLOR_HIGHLIGHT_TEXT]Priest[COLOR_REVERT] specialists. It also increases the odds of its city generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</Spanish>
+		<English>The [COLOR_BUILDING_TEXT]Borobudur[COLOR_REVERT] allows you to hire three [COLOR_HIGHLIGHT_TEXT]Priest[COLOR_REVERT] specialists and adds additional production to all of your [COLOR_HIGHLIGHT_TEXT]Priest[COLOR_REVERT] specialists. It also increases the odds of its city generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</English>
```
Wikipedia usage suggests no article in conjunction with Borobudur. It's treated like a place name.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -201,13 +217,21 @@
-		<German>The [COLOR_BUILDING_TEXT]Temple of Kukulkan[COLOR_REVERT] increases all cities' defenses by 25%%. It improves the odds of its city generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</German>
-		<Italian>The [COLOR_BUILDING_TEXT]Temple of Kukulkan[COLOR_REVERT] increases all cities' defenses by 25%%. It improves the odds of its city generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</Italian>
-		<Spanish>The [COLOR_BUILDING_TEXT]Temple of Kukulkan[COLOR_REVERT] increases all cities' defenses by 25%%. It improves the odds of its city generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</Spanish>
+		<English>The [COLOR_BUILDING_TEXT]Temple of Kukulkan[COLOR_REVERT] adds extra food to all rainforest tiles in its city. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</English>
```
... all rainforest tiles worked by its city.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -341,6 +381,14 @@
 		<Spanish>The Great Bath</Spanish>
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_HELP_GREAT_BATH</Tag>
```
This wonder does not exist anymore, I think we can get rid of the text.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -359,11 +407,11 @@
-		<German>The [COLOR_BUILDING_TEXT]Great Cothon[COLOR_REVERT] add a bonus production to every water tile in the city in which it's constructed. It increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</German>
-		<Italian>The [COLOR_BUILDING_TEXT]Great Cothon[COLOR_REVERT] add a bonus production to every water tile in the city in which it's constructed. It increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</Italian>
-		<Spanish>The [COLOR_BUILDING_TEXT]Great Cothon[COLOR_REVERT] add a bonus production to every water tile in the city in which it's constructed. It increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</Spanish>
+		<English>The [COLOR_BUILDING_TEXT]Great Cothon[COLOR_REVERT] add a bonus production to every water tile in its city. It increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</English>
```
... worked by its city.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -469,6 +493,22 @@
+	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_HIMEJI_CASTLE_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Himeji Castle[COLOR_REVERT] makes city defence increases culture output in all cities and makes all fully fortified units in its provide +1 culture. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Statesman[COLOR_REVERT].</English>
```
+1 culture per level.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -485,6 +525,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_ISHTAR_GATE_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Ishtar Gate[COLOR_REVERT] allows you to hire a specialist of each type. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
one specialist of each type

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -517,6 +565,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_KHAJURAHO_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Khajuraho[COLOR_REVERT] adds additional food to all buildings of your state religion. It also increases its city's chances of generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</English>
```
I would also go without an article here.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -589,6 +645,22 @@
 		<Italian>La Mezquita</Italian>
 		<Spanish>La Mezquita</Spanish>
 	</TEXT>
+	<!--<TEXT>
```
Why not remove the commented text?

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -755,11 +795,11 @@
-		<German>The [COLOR_BUILDING_TEXT]Grand Canal[COLOR_REVERT] adds extra commerce to all river tiles in your city. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Engineer[COLOR_REVERT].</German>
-		<Italian>The [COLOR_BUILDING_TEXT]Grand Canal[COLOR_REVERT] adds extra commerce to all river tiles in your city. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Engineer[COLOR_REVERT].</Italian>
-		<Spanish>The [COLOR_BUILDING_TEXT]Grand Canal[COLOR_REVERT] adds extra commerce to all river tiles in your city. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Engineer[COLOR_REVERT].</Spanish>
+		<English>The [COLOR_BUILDING_TEXT]Grand Canal[COLOR_REVERT] adds extra commerce to all river tiles in its city. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Engineer[COLOR_REVERT].</English>
```
worked by its city

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -865,13 +873,29 @@
+		<Italian>[ICON_BULLET]No open borders required for trade</Italian>
+		<Spanish>[ICON_BULLET]No open borders required for trade</Spanish>
+	</TEXT>
+	<!--<TEXT>
```
remove

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -865,13 +873,29 @@
 		<Italian>Torre di Porcellana</Italian>
 		<Spanish>Torre de porcelana</Spanish>
 	</TEXT>
+	<TEXT>
```
This text key is not used anymore afaik.

MeuhV1 replied:
It is still used by BuildingInfos.xml. And in the DLL, that bonus still seems to be in effect.

dguenms replied:
Okay.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -865,13 +873,29 @@
-		<German>The [COLOR_BUILDING_TEXT]Porcelain Tower[COLOR_REVERT] gives 2 free [COLOR_HIGHLIGHT_TEXT]Merchant[COLOR_REVERT] specialists and removes the open border requirement for trade routes. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</German>
-		<Italian>The [COLOR_BUILDING_TEXT]Porcelain Tower[COLOR_REVERT] gives 2 free [COLOR_HIGHLIGHT_TEXT]Merchant[COLOR_REVERT] specialists and removes the open border requirement for trade routes. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</Italian>
-		<Spanish>The [COLOR_BUILDING_TEXT]Porcelain Tower[COLOR_REVERT] gives 2 free [COLOR_HIGHLIGHT_TEXT]Merchant[COLOR_REVERT] specialists and removes the open border requirement for trade routes. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</Spanish>
+		<English>The [COLOR_BUILDING_TEXT]Porcelain Tower[COLOR_REVERT] adds additional food to all of your [COLOR_HIGHLIGHT_TEXT]Statesman[COLOR_REVERT] specialists and removes the open border requirement for trade routes. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</English>
```
See above.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1193,6 +1201,22 @@
+	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_TOPKAPI_PALACE_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Topkapi Palace[COLOR_REVERT] increases the amount of culture converted on city conquest and grants extra initial production in the conquered city. It also increase their city's chances of generating a [COLOR_UNIT_TEXT]Great Statesman[COLOR_REVERT].</English>
```
It does no longer affect the amount of culture converted.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1291,19 +1291,19 @@
-		<German>The [COLOR_BUILDING_TEXT]University of Sankore[COLOR_REVERT] gives all buildings of your state religion a research bonus. It increases its city's chances of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</German>
-		<Italian>The [COLOR_BUILDING_TEXT]University of Sankore[COLOR_REVERT] gives all buildings of your state religion a research bonus. It increases its city's chances of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</Italian>
-		<Spanish>The [COLOR_BUILDING_TEXT]University of Sankore[COLOR_REVERT] gives all buildings of your state religion a research bonus. It increases its city's chances of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</Spanish>
+		<English>The [COLOR_BUILDING_TEXT]University of Sankore[COLOR_REVERT] adds extra commerce to all desert tiles in all cities. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</English>
```
for all cities

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1393,6 +1377,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_GONDESHAPUR_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Gondeshapur[COLOR_REVERT] gives all eligible units built in this city the [COLOR_HIGHLIGHT_TEXT]Medic I[COLOR_REVERT] promotion and adds additional food to all of your [COLOR_HIGHLIGHT_TEXT]Scientist[COLOR_REVERT] specialists. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</English>
```
No article here as well.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1409,6 +1401,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_PRAMBANAN_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Prambanan[COLOR_REVERT] lets all your cities store extra food after growth. It also increases its city's chance of generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</English>
```
No article here as well.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1417,6 +1417,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_CHEOMSEONGDAE_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Cheomseongdae[COLOR_REVERT] increases its city's happiness output from using the science slider. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</English>
```
No article here as well.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1425,6 +1433,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_NALANDA_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Nalanda University[COLOR_REVERT] adds additional science to all buildings of your state religion. It also increases its city's chances of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</English>
```
No article here as well.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1481,6 +1521,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_SAINT_SOPHIA_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Saint Sophia's Cathedral[COLOR_REVERT] adds additional food to all of your [COLOR_HIGHLIGHT_TEXT]Merchant[COLOR_REVERT] specialists. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Statesman[COLOR_REVERT].</English>
```
No article here as well.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1489,6 +1537,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_SANTA_MARIA_DEL_FIORE_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Santa Maria del Fiore[COLOR_REVERT] increase the rate of its city Great People Points. It also increases its city's chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
No article here as well.

dguenms replied:
increases

dguenms replied:
points (lower case)

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1521,6 +1593,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_SAINT_THOMAS_CHURCH_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Saint Thomas Church[COLOR_REVERT] increases the happiness of all of your cities. It increases its city's chances of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
No article here as well.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1585,6 +1689,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_SHALIMAR_GARDENS_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Shalimar Gardens[COLOR_REVERT] provides happiness for excessive health in its city. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
provide

dguenms replied:
excess health

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1569,6 +1665,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_POTALA_PALACE_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Potala Palace[COLOR_REVERT] provides a Great People point for every hill tile in the city's radius. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</English>
```
No article here as well.

dguenms replied:
in the city radius

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1681,6 +1849,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_BELL_ROCK_LIGHTHOUSE_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Bell Rock Lighthouse[COLOR_REVERT] provides a free trade route in all coastal cities. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</English>
```
No article here as well.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1737,6 +1945,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_SAGRADA_FAMILIA_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Sagrada Familia[COLOR_REVERT] adds additional production to all of your [COLOR_HIGHLIGHT_TEXT]Artist[COLOR_REVERT] specialists and increases the efficiency of building commerce. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
No article here as well.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1745,6 +1961,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_DUJIANGYAN_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Dujiangyan[COLOR_REVERT] adds extra food to all river tiles in its city. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Statesman[COLOR_REVERT].</English>
```
No article.

dguenms replied:
worked by its city

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1809,6 +2065,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_HARBOUR_OPERA_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Harbour Opera[COLOR_REVERT] provides food for excessive happiness in its city. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
per excess happiness

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1819,11 +2083,19 @@
-		<German>[ICON_BULLET]+2[ICON_FOOD] +2[ICON_COMMERCE] per Desert tile</German>
-		<Italian>[ICON_BULLET]+2[ICON_FOOD] +2[ICON_COMMERCE] per Desert tile</Italian>
-		<Spanish>[ICON_BULLET]+2[ICON_FOOD] +2[ICON_COMMERCE] per Desert tile</Spanish>
+		<English>[ICON_BULLET]Requires Desert in city radius[NEWLINE][ICON_BULLET]Desert tiles in city radius yield at least 2[ICON_FOOD] and 2[ICON_COMMERCE]</English>
```
Desert tiles worked by the city

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1819,11 +2083,19 @@
+	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_BURJ_KHALIFA_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Burj Khalifa[COLOR_REVERT] increases the yield of desert tiles. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Engineer[COLOR_REVERT].</English>
```
No article.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1819,11 +2083,19 @@
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_BURJ_KHALIFA_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Burj Khalifa[COLOR_REVERT] increases the yield of desert tiles. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Engineer[COLOR_REVERT].</English>
+		<French>The [COLOR_BUILDING_TEXT]Burj Khalifa[COLOR_REVERT] increases the yield of desert tiles. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Engineer[COLOR_REVERT].</French>
```
food and commerce yield

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1833,6 +2105,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_HUBBLE_SPACE_TELESCOPE_STRATEGY</Tag>
+		<English>[COLOR_BUILDING_TEXT]Hubble Space Telescope[COLOR_REVERT] adds additional science to all of your settled satellites. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</English>
```
But I'd put an article here :)

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1849,6 +2129,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_CHANNEL_TUNNEL_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Channel Tunnel[COLOR_REVERT] increases increases the yield of all trade routes with civilizations with friendly relation. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Engineer[COLOR_REVERT].</English>
```
"increases increases"

dguenms replied:
friendly relations

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1865,6 +2153,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_DELTA_WORKS_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Delta Works[COLOR_REVERT] adds extra food, production and commerce to all water tiles in its city. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</English>
```
Either no article or "The Delta Works add" ... I prefer the former

dguenms replied:
worked by its city

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1897,6 +2201,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_ORIENTAL_PEARL_TOWER_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Oriental Pearl Tower[COLOR_REVERT] provides 2 science per population of the city in which it's built. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT]</English>
```
per population in its city

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1913,6 +2225,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_GARDENS_BY_THE_BAY_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Gardens by the Bay[COLOR_REVERT] provides commerce for excessive health in its city. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
excess health

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1929,6 +2249,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_BLETCHLEY_PARK_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Bletchley Park[COLOR_REVERT] decreases the change your spies are being detected and increases the chance to detect enemy spies. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</English>
```
No article.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1953,6 +2281,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_MOTHERLAND_CALLS_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Motherland Calls[COLOR_REVERT] drafts an unit when a unit of level 3 or higher is lost. It also increases its city's chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
No article.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -1969,6 +2305,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_BERLAYMONT_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Berlaymont[COLOR_REVERT] increases the limit of defensive pacts you can sign and provides provides 1 free specialist per signed defensive pact. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Statesman[COLOR_REVERT].</English>
```
No article.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2009,6 +2377,22 @@
+		<Italian>The [COLOR_BUILDING_TEXT]Shwedagon Paya[COLOR_REVERT] doubles the construction speed of the buildings of your state religion. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</Italian>
+		<Spanish>The [COLOR_BUILDING_TEXT]Shwedagon Paya[COLOR_REVERT] doubles the construction speed of the buildings of your state religion. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</Spanish>
+	</TEXT>
+	<!--<TEXT>
```
Remove commented text.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2026,20 +2410,28 @@
-		<Italian>[ICON_BULLET]At most 1[ICON_UNHAPPY] from temporary effects</Italian>
-		<Spanish>[ICON_BULLET]At most 1[ICON_UNHAPPY] from temporary effects</Spanish>
+		<Tag>TXT_KEY_BUILDING_NOTRE_DAME_STRATEGY</Tag>
+		<English>[COLOR_BUILDING_TEXT]Notre Dame[COLOR_REVERT] limits the amount of required temples to build Cathedrals to three. It also increases its city's chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
reduces the number of temples required to ...

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2057,6 +2449,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_AL_KHAZNEH_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Al Khazneh[COLOR_REVERT] provides a free trade route in all cities. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</English>
```
No article.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2065,6 +2465,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_ALAMUT_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Alamut Fortress[COLOR_REVERT] grants much experience to [COLOR_UNIT_TEXT]Espionage Units[COLOR_REVERT]. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Statesman[COLOR_REVERT].</English>
```
No article.

dguenms replied:
much ->  a lot of

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2081,6 +2489,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_MOUNT_ATHOS_STRATEGY</Tag>
+		<English>[COLOR_BUILDING_TEXT]Mount Athos[COLOR_REVERT] doubles the Great People Points from all buildings. It also increases its city's chance of generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</English>
```
No article.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2113,6 +2537,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_GUADALUPE_BASILICA_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Guadalupe Basilica[COLOR_REVERT] produces one gold each turn for every city on the continent of its city that practises [COLOR_HIGHLIGHT_TEXT]Catholicism[COLOR_REVERT]. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT].</English>
```
No article.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2187,11 +2651,19 @@
-		<German>[ICON_BULLET]Other civilisation are more willing to enter defensive pacts[NEWLINE][ICON_BULLET]Shares golden ages of defensive pact partners</German>
-		<Italian>[ICON_BULLET]Other civilisation are more willing to enter defensive pacts[NEWLINE][ICON_BULLET]Shares golden ages of defensive pact partners</Italian>
-		<Spanish>[ICON_BULLET]Other civilisation are more willing to enter defensive pacts[NEWLINE][ICON_BULLET]Shares golden ages of defensive pact partners</Spanish>
+		<English>[ICON_BULLET]Other civilization are more willing to enter defensive pacts[NEWLINE][ICON_BULLET]Shares golden ages of defensive pact partners</English>
```
enter into defensive pacts

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2187,11 +2651,19 @@
+	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_AMBER_ROOM_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Amber Room[COLOR_REVERT] increases the willingness of other civilizations to enter defensive pact and shars golden ages with defensive pact partners. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
enter into defensive pacts

dguenms replied:
shares

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2203,19 +2675,19 @@
-		<Italian>[ICON_BULLET]Units are completely healed when promoted</Italian>
-		<Spanish>[ICON_BULLET]Units are completely healed when promoted</Spanish>
+		<Tag>TXT_KEY_BUILDING_PALACE_OF_NATIONS_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Palace of Nations[COLOR_REVERT] defends your cities from being demanded in Congress and other civilizations are more willing to approve your demands in Congress. It will cause Congresses to occur every four turns. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Statesman[COLOR_REVERT].</English>
```
It makes Congresses occur every four turns.

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2313,6 +2825,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_NOBEL_PRIZE_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Nobel Prize[COLOR_REVERT] provides Great People Points when a Great Person is born in a pleased or friendly Civilization. It also increases its city's chance of generating a [COLOR_UNIT_TEXT]Great Statesman[COLOR_REVERT].</English>
```
civilization

dguenms replied:
in a civilization with pleased or friendly relations

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2385,6 +2929,14 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_CHAPULTEPEC_CASTLE_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Chapultepec Castle[COLOR_REVERT] grants 1 experience to units build in its city per culture level of its city. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Statesman[COLOR_REVERT].</English>
```
grants 1 experience per culture level to units built in its city

dguenms replied:
built

dguenms replied:
No article

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2401,6 +2953,30 @@
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_FRONTENAC_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Chateau Frontenac[COLOR_REVERT] creates a free [COLOR_BUILDING_TEXT]Railway Station[COLOR_REVERT] in every city and increases the building speed for railroads. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</English>
```
No article

dguenms commented on Assets/XML/Text/World Wonders.xml:
```diff
@@ -2401,6 +2953,30 @@
+	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_CRISTO_REDENTOR_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Cristo Redentor[COLOR_REVERT] allows its builder to switch [COLOR_HIGHLIGHT_TEXT]Religions[COLOR_REVERT] or [COLOR_HIGHLIGHT_TEXT]Civics[COLOR_REVERT] instantly once per turn, with no anarchy. It also increases the chances of a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT] appearing.</English>
```
No article

MeuhV1 replied:
This is the original text from BTS. I only changed the Great Person type.

dguenms commented on CvGameCoreDLL/CvCity.cpp:
```diff
@@ -2867,14 +2867,22 @@ int CvCity::getProductionExperience(UnitTypes eUnit)
 
 			//Leoreth: Harmandir Sahib effect
             if (GET_PLAYER(getOwnerINLINE()).isHasBuilding((BuildingTypes)HARMANDIR_SAHIB))
+			{
```
The indentation here looks wrong.

MeuhV1 replied:
This is only the github viewer. The file itself is correct. (I guess it is related to spaces vs. tabs)

dguenms replied:
Well spaces and tabs are different, we should be consistent. Looks like tabs is the default in this file.

dguenms commented on CvGameCoreDLL/CvPlot.cpp:
```diff
@@ -6687,7 +6687,10 @@ int CvPlot::calculateNatureYield(YieldTypes eYield, TeamTypes eTeam, bool bIgnor
+			if (eYield == YIELD_FOOD) return 2;
+			if (eYield == YIELD_PRODUCTION) return 1;
+		}
 		else
```
I would prefer curly brackets here as well.

dguenms commented on CvGameCoreDLL/CvPlot.cpp:
```diff
@@ -6908,6 +6911,26 @@ int CvPlot::calculateYield(YieldTypes eYield, bool bDisplay) const
 
 	iYield = calculateNatureYield(eYield, ((ePlayer != NO_PLAYER) ? GET_PLAYER(ePlayer).getTeam() : NO_TEAM));
+	
+	// Leoreth + Merijn: Burj Khalifa effect
```
Have you checked if the changed output is displayed in the city screen even without working the tile?

MeuhV1 replied:
It does.

If a tile can be worked by multiple cities, it will show the yield of the city that "controls" that tile. (Even if it is not currently worked)

dguenms reviewed (commented):
I assume you are not yet done addressing comments?

MeuhV1 commented:
I did adress all thing you mentioned. But somehow this page hasn't updated. The tab "changed files" does show the updates. For example, I did change the tab-space thing as well as the curly brackets.

EDIT: O wait. I see that I indeed missed some. I click on "show more" and I assumed that all comments were shown. But there were still some comments that were not loaded yet. I will fix them soon.

Commit:
Strategy texts for all new wonders
Changed Burj Khalifa effect: Desert tiles in city radius base yield is at least 2 food and 2 commerce
Some wonders provide different GPP:
The Cristo Redentor: Great Engineer -> Great Prophet
Eiffel Tower: Great Merchant -> Great Artist
Sagrada Familia: Great Engineer -> Great Artist
Salt Cathedral: Great Engineer -> Great Merchant
Shalimar Gardens: Great Engineer -> Great Artist
Trafalgar Square: Great Merchant -> Great Statesman
Some minor code updates

MeuhV1 commented:
Now everything should be adressed.
